src/NuGet.Core/NuGet.Packaging/Signing/Signatures/SignatureVerifySettings.cs

```diff
@@ -4,20 +4,24 @@
 namespace NuGet.Packaging.Signing
 {
     /// <summary>
-    /// Settings to customize Signature.Verify behavior. 
+    /// Settings to customize Signature.Verify behavior.
     /// </summary>
     public sealed class SignatureVerifySettings
     {
         /// <summary>
-        /// Treat any issue as fatal.
+        /// Allow packages with signatures that do not conform to the specification.
         /// </summary>
-        /// <remarks>If set true, any issue will be an error instead of a warning</remarks>
-        public bool TreatIssuesAsErrors { get; }
+        public bool AllowIllegal { get; }
 
         /// <summary>
         /// Specifies that a signing certificate's chain that chains to an untrusted root is allowed
         /// </summary>
-        public bool AllowUntrustedRoot { get; }
+        public bool AllowUntrusted { get; }
+
+        /// <summary>
+        /// Indicates if untrusted status should be reported.
```

Comment:
>reported [](start = 52, length = 8)

Maybe we should mention that these are reported to the `issues` parameter of `Signature.Verify`?